docs/distributed/README.md

```diff
@@ -34,7 +34,7 @@ If you're aware of stand-alone MinIO set up, the process remains largely the sam
 
 ### 1. Prerequisites
 
-Install MinIO - [MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide).
+Install MinIO - [MinIO Quickstart Guide](https://min.io/docs/minio/kubernetes/upstream/index.html#quickstart-for-kubernetes).
```

Comment:
Lets split this into two links:

- One for Tenants on Kubernetes
- One for a deployment on Linux

The quickstart isn't distributed, so that doesn't help here.

Reply:
Really we can probably cull this entire page and point to the docs, and eventually re-use it for internal documentation and engineering guidance.

Reply:
Rewrote the section to split to either install on Kubernetes or install in distributed mode on Linux.